projects/artemis/index.js

```diff
@@ -4,12 +4,47 @@ const { masterchefExports, } = require("../helper/unknownTokens")
 const token = ADDRESSES.harmony.MIS;
 const masterchef = "0x59c777cd749b307be910f15c54a3116ff88f9706";
 
-module.exports = masterchefExports({
-  chain: 'harmony',
-  useDefaultCoreAssets: true,
-  masterchef,
-  nativeToken: token,
-  blacklistedTokens: [
-    '0xed0b4b0f0e2c17646682fc98ace09feb99af3ade', // RVRS bad pricing
-  ]
-})
\ No newline at end of file
+const sdk = require('@defillama/sdk');
+async function goatTvl({block, chain}) {
```

Comment:
is chain defined when you test this code? We have pricing data for all of these goat assets so there might be an issue in your code

Reply:
fixed, thanks.